sdk/attestation/azure-security-attestation/src/attestation_client.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/attestation/attestation_client.hpp"
+#include <shared_mutex>
+
+#include "azure/core/base64.hpp"
+#include "azure/core/http/policies/policy.hpp"
+#include "azure/core/internal/http/pipeline.hpp"
```

Comment:
change quotes for diamond braces <>

Reply:
You told me in an email that the convention was that all files in the same repo used "" and not <>.

Azure/Core is in the same repo as Azure/Attestation, so shouldn't it use the "" convention?